mscore/prefsdialog.ui

```diff
@@ -2703,6 +2703,25 @@
        <string>I/O</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_8">
+       <item>
+        <layout class="QGridLayout" name="gridLayout_16"/>
+       </item>
+       <item>
+        <widget class="Ms::RadioButtonGroupBox" name="coreAudioDriver">
+         <property name="minimumSize">
+          <size>
+           <width>0</width>
+           <height>30</height>
+          </size>
+         </property>
+         <property name="title">
+          <string>Core Audio</string>
+         </property>
+         <property name="checkable">
+          <bool>true</bool>
+         </property>
+        </widget>
+       </item>
```

Comment:
Qt Creator made all sorts of unnecessary changes to this file. This is all that is needed. That and adding a tabstop for the coreAudioDriver widget towards the end of the file.

Reply:
that tabstop got lost now

Reply:
It didn't get lost, it was never added in the first place.

Reply:
Can you add it then?